pipelines/ci-sit-kubernetes-pipeline.yml

```diff
@@ -903,3 +991,67 @@ jobs:
     input_mapping: {
       docker-image-resource: fieldwork-adapter-docker-latest,
       kubernetes-repo: census-rm-kubernetes-microservices-repo}
+
+- name: "SIT Deploy Notify Processor"
+  serial: true
+  serial_groups: [sit-notify-processor]
+  plan:
+  - get: census-rm-terraform
+    trigger: true
+    passed: ["CI Acceptance Tests"]
+  - get: census-rm-kubernetes-microservices-repo
+    trigger: true
+    passed: ["CI Acceptance Tests"]
+  - get: notify-processor-docker-latest
+    trigger: true
+    passed: ["CI Acceptance Tests"]
+    params:
+      skip_download: true
+  - get: census-rm-deploy
+  - task: apply-deployment
+    file: census-rm-deploy/tasks/kubectl-apply-deployment.yml
+    on_failure: *slack_failure_alert_sit
+    params:
+      SERVICE_ACCOUNT_JSON: ((gcp.service_account_json))
+      GCP_PROJECT_NAME: ((sit-gcp-project-name))
+      KUBERNETES_CLUSTER: ((sit-kubernetes-cluster-name))
+      KUBERNETES_DEPLOYMENT_NAME: notify-processor
+      KUBERNETES_SELECTOR: app=notify-processor
+      KUBERNETES_FILE_PATH: kubernetes-repo/microservices
+      KUBERNETES_FILE_PREFIX: notify-processor
+      WAIT_UNTIL_AVAILABLE_TIMEOUT: 200s
+    input_mapping: {
+      docker-image-resource: notify-processor-docker-latest,
+      kubernetes-repo: census-rm-kubernetes-microservices-repo}
+
+- name: "SIT Deploy Notify Stub"
```

Comment:
we won't need the stub in SIT because we're going to use the real Gov.Notify with a test API key. Also, we don't do automated acceptance testing in SIT. My mistake.